index.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="queries.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@200;300;400;500;600;700&family=Lato:ital,wght@0,100;0,300;0,400;0,700;1,100;1,300;1,400;1,700&family=Poppins:ital,wght@0,200;0,300;0,400;0,700;1,100;1,200;1,300;1,400;1,700&family=Roboto:ital,wght@0,700;1,700&display=swap" rel="stylesheet">
+    <title>Agropreneurs</title>
+</head>
+<body>
+    <header>
+        <div class="socials">
+            <ul>
+                <li><a href=""><img src="images/images (3).png" alt="facecbook Logo"></a></li>
+                <li><a href=""><img src="images/download.png" alt="twitter Logo"></a></li>
+                <li><a href="">Logout</a></li>
+                <li><a href="">English</a></li>
+                <li><a href="">Bini</a></li>
+            </ul>
+
+        </div>
+        <nav class="nav_items">
+            <div class="Logo">
+                <a href="index.html"><img src="images/images (4).png" alt="Website Logo"></a>
+            </div>
+            <ul class="pop-up" id="pop-up">
+                
+                <li>
+                    <a href="About.html" class="about" id="nav">About</a>
+                </li>
+                <li>
+                    <a href="" class="a" id="nav1">Program</a>
+                </li>
+                <li>
+                    <a href="" class="a" id="nav2">Join</a>
+                </li>
+                <li>
+                    <a href="" class="a" id="nav3">Sponsor</a>
+                </li>
+                <li>
+                    <a href="" class="a" id="nav4">News</a>
+                </li>
+                <li>
+                    <a href="" class="about program" id="nav5">AgCampaign</a>
+                </li>
+                <img src="images/2794770.png" alt="close button" class="img" id="close">
+            </ul>
```

Comment:
- [ ] Good job adding this layout as required :clap: , but there is one thing that will make perfect your project. I noticed that when you click on one of these nav-link, it should drive the users to the corresponding section. Kindly update add the correct `href` to improve the user experience :100: 

![Screenshot from 2023-01-12 13-02-27](https://user-images.githubusercontent.com/80458199/212050126-0427fd56-e568-4da9-8674-4bbb2886b278.png)

You could do for example:

```bash
<a href="#section-id" class="a" id="nav4">News</a>
```
```bash
<section class="Talkers" id="section-id">
```

Look at how I use `section-id` in the example above